htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -309,6 +310,11 @@
 			echo '<input type="hidden" name="pbq" id="pbq" value="">';
 			echo '</span>';
 		}
+
+		if (!empty($conf->global->MAIN_VIEW_LINE_NUMBER)) {
```

Comment:
I would suggest to use another constant like
MAIN_ADD_LINE_NUMBER
and keep MAIN_VIEW_LINE_NUMBER for current behaviour (read only the column)

Reply:
Hi @eldy, i did it and created hidden conf "MAIN_ADD_LINE_AT_POSITION"